.github/workflows/ExtensionTemplate.yml

```diff
@@ -101,6 +110,11 @@ jobs:
           cd duckdb
           git checkout ${{ matrix.duckdb_version }}
 
+      - name: Setup vcpkg
+        uses: lukka/run-vcpkg@v11
+        with:
+          vcpkgGitCommitId: 501db0f17ef6df184fcdbfbe0f87cde2313b6ab1
```

Comment:
This can be potentially a environment variable / github.event.input.

Reply:
I haven't found a way to do this, `github.event.input` doesn't work because: 
> The inputs context contains input properties passed to an action, to a reusable workflow, or to a manually triggered workflow.
This is neither. Also env variables are not accessible here so idk?